docs/UsingSpiffdemo/Basic_Tools.md

```diff
@@ -9,33 +9,22 @@ capabilities.
 Here are fundamental blocks of our system:
 
 ## Displaying Content
-The Spiffarena platform offers powerful features for displaying 
+The SpiffArena platform offers powerful features for displaying 
 content  within your BPMN processes.
-Follow the steps below to effectively display content in your 
-processes:
 
 In this guide, we'll explore how to effectively display content in 
-Spiffarena, providing a seamless user experience and engaging 
+SpiffArena, providing a seamless user experience and engaging 
 presentation of information. 
 
 Let's dive in and learn the key aspects of displaying content within 
 your workflows.
 
-### Markdown: Formatting Content
-Markdown is a powerful tool for formatting and styling your content in 
-Spiffarena. With Markdown, you can easily add headings, lists, tables, 
+- **Markdown**: Markdown is a powerful tool for formatting and styling your content in 
+SpiffArena. With Markdown, you can easily add headings, lists, tables, 
 hyperlinks, and more to enhance the readability and visual appeal of 
 your displayed content. 
 
-### Jinja Templating: Dynamic Content Generation
-
-Jinja Templating in Spiffarena empowers you to generate dynamic 
-content based on variables, conditions, and logic. By leveraging 
-Jinja's syntax and functionality, you can customize your content to 
-adapt to specific scenarios, display personalized information, or 
-perform calculations based on collected data.
-
-### Combining Markdown and Jinja: Unleashing the Power
+- **Jinja Templating**: Jinja Templating in SpiffArena empowers you to generate dynamic content based on variables, conditions, and logic. By leveraging Jinja's syntax and functionality, you can customize your content to adapt to specific scenarios, display personalized information, or perform calculations based on collected data.
```

Comment:
Thanks for putting sentences on their own lines. Please move the second sentence (the one starting with "By leveraging") to its own line as well.